src/reducers/sources.js

```diff
@@ -37,13 +37,14 @@ export type SourcesState = {
 const State = makeRecord(({
   sources: I.Map(),
   selectedLocation: undefined,
-  pendingSelectedLocation: undefined,
+  pendingSelectedLocation: prefs.pendingSelectedLocation,
```

Comment:
Hi @jasonLaster 

Here is a bug that I talked about with pendingSelectedLocation:

![lhexal3nx5](https://cloud.githubusercontent.com/assets/4562118/21742386/d84508f8-d528-11e6-81ac-cf258c457547.gif)

I think we may have to change the type of pendingSelectedLocation because pendingSelectedLocation only contains the most recent selected tab's url.